### ISSUE TITLE
Comment at the end throw an unintended error

### ISSUE THREAD

**Issue by hrdtbs** [myuon/eslint-plugin-css-reorder#11]
```tsx
const resetStyles = css`
  /* # =================================================================
      # Accessibility
      # ================================================================= */

  /* Specify the progress cursor of updating elements */
  [aria-busy="true"] {
    cursor: progress;
  }

  /* Specify the pointer cursor of trigger elements */
  [aria-controls] {
    cursor: pointer;
  }

  /* Specify the unstyled cursor of disabled, not-editable, or otherwise inoperable elements */
  [aria-disabled="true"] {
    cursor: default;
  } /* # =================================================================
  # Global selectors
  # ================================================================= */ /* resset.dev • v5.0.2 */ /* @see https://github.com/filipelinhares/ress/blob/master/ress.css */
`;
```

**Comment by hrdtbs** [comment]
Why...
<img width="783" alt="スクリーンショット 2022-05-06 18 40 23" src="https://user-images.githubusercontent.com/30747709/167107547-de70eef9-6042-4232-8b83-3d50efc14390.png">

**Comment by hrdtbs** [comment]
Perhaps this is the cause...
` /* resset.dev • v5.0.2 */ /* @see https://github.com/filipelinhares/ress/blob/master/ress.css */`

**Comment by myuon** [comment]
Should should be fixed in v0.5.1
